constants/types/Post.d.ts

```diff
@@ -0,0 +1,24 @@
+interface PostAudio {
```

Comment:
Vad innebär det med en .d-fil? :)

Reply:
Lagt in en kommentar i koden. https://medium.com/jspoint/typescript-type-declaration-files-4b29077c43